benchexec/tools/kissat.py

```diff
@@ -0,0 +1,48 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for Kissat SAT Solver.
+    URL: http://fmv.jku.at/kissat/
+    """
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("kissat", subdir="build")
+
+    def name(self):
+        return "Kissat"
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def cmdline(self, executable, options, task, rlimits):
+        return [executable] + options + list(task.input_files_or_identifier)
```

Comment:
I guess that Kissat always needs exactly one input file? In this case, just use `[task.single_input_file]` as last part of the command line.

Reply:
Done.